heron/common/src/cpp/config/topology-config-vars.cpp

```diff
@@ -39,6 +39,11 @@ const sp_string TopologyConfigVars::TOPOLOGY_ENABLE_MESSAGE_TIMEOUTS =
 const sp_string TopologyConfigVars::TOPOLOGY_ENABLE_ACKING = "topology.acking";
 const sp_string TopologyConfigVars::TOPOLOGY_CONTAINER_CPU_REQUESTED = "topology.container.cpu";
 const sp_string TopologyConfigVars::TOPOLOGY_CONTAINER_RAM_REQUESTED = "topology.container.ram";
+const sp_string TopologyConfigVars::TOPOLOGY_STATEFUL_ENABLED = "topology.stateful.enabled";
+const sp_string TopologyConfigVars::TOPOLOGY_STATEFUL_CHECKPOINT_INTERVAL_SECONDS =
+                            "topology.stateful.checkpoint.interval.seconds";
+const sp_string TopologyConfigVars::TOPOLOGY_STATEFUL_START_CLEAN = "topology.stateful.start.clean";
+const sp_string TopologyConfigVars::TOPOLOGY_EXACTLYONCE_ENABLED = "topology.exactlyonce.enabled";
```

Comment:
We could certainly clean that up in a later pr. However one good thing is that all of these methods are abstracted out by the config helpers. So when we make that change, its just a matter of making the change in this class rather than in any other component.